drivers/timer/mcux_os_timer.c

```diff
@@ -16,7 +16,9 @@
 #include <zephyr/drivers/counter.h>
 #include <zephyr/pm/pm.h>
 #include "fsl_ostimer.h"
+#ifndef CONFIG_SOC_SERIES_MCXN23X
```

Comment:
You could instead use this
`#if (DT_INST_PROP(0, wakeup_source))`

Reply:
@mmahadevan108 is this a "could" or "should"? Not clear to me why you would use this over the other?

Reply:
Just to avoid platform specific defines

Reply:
it looks like it's because MCXN236 has no fsl_power.h, for some reason, I'm not sure why though